api/dataservices/interface.go

```diff
@@ -191,6 +215,21 @@ type (
 		DeleteAPIKey(ID portainer.APIKeyID) error
 		GetAPIKeysByUserID(userID portainer.UserID) ([]portainer.APIKey, error)
 		GetAPIKeyByDigest(digest []byte) (*portainer.APIKey, error)
+		GetIdentifier() int
+		SetIdentifier(id int) error
+	}
+
+	// ScheduleService represents a service for managing scheduled jobs
+	ScheduleService interface {
```

Comment:
is this supposed to be part of this PR?

Reply:
Discovered this should have been in the interface but was missing.

Reply:
Oh maybe this needs to be removed.  Not sure its part of CE.